test/ClangModules/AppKit_test.swift

```diff
@@ -23,15 +23,15 @@ func test(_ URL: NSURL, controller: NSDocumentController) {
 
 extension NSBox {
   func foo() {
-    print("abc") // expected-warning {{use of 'print' treated as a reference to instance method in class 'NSView'}}
+    print("abc" as NSString) // expected-warning {{use of 'print' treated as a reference to instance method in class 'NSView'}}
     // expected-note@-1 {{use 'self.' to silence this warning}} {{5-5=self.}}
     // expected-note@-2 {{use 'Swift.' to reference the global function}} {{5-5=Swift.}}
   }
 }
 
 class MyView : NSView {
   func foo() {
-    print("abc") // expected-warning {{use of 'print' treated as a reference to instance method in class 'NSView'}}
+    print("abc" as NSString) // expected-warning {{use of 'print' treated as a reference to instance method in class 'NSView'}}
```

Comment:
We'll still infer type String for "abc", and then raise an invalid conversion error. (We do this because String is the favored type for all string literals.) I'll see if I can do something about this case.